TurtleShroom Productions V

TurtleShroom opened the pull request.

TurtleShroom commented:
@loup99 / @EmperorEnlil 

I don't even know why I bother. I wrote the Event and immediately broke the entire Mod. IT SOMEHOW BROKE THE DECISIONS, BUT I EDITED A FASTING EVENT. I am such a retarded idiot. I'm going to try again and get this daggum thing to work, but I'm just going to go ahead and call you in. I am so humiliated right now.

TurtleShroom commented:
![untitled](https://cloud.githubusercontent.com/assets/17663470/18970389/21bbc77e-865f-11e6-902b-5e9c10e7ac44.png)

All I can do is scream.

TurtleShroom commented:
@loup99 / @EmperorEnlil 

![untitled](https://cloud.githubusercontent.com/assets/17663470/18970490/7d0abaae-865f-11e6-80f9-02b730a2c402.png)

_**HOW DID I BREAK THE DECISIONS MENU. I DIDN'T EVEN EDIT A FASTING DECISION. WHAT THE HECK DID I DO. WHY DO I ALWAYS FAIL AT EVENTS.  WHY DO I BREAK EVEYRTHING WHEN I WRITE A FASTING EVENT**_

_**KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME KILL ME**_

I HATE THIS

TurtleShroom commented:
@loup99 / @EmperorEnlil 

![untitled](https://cloud.githubusercontent.com/assets/17663470/18971259/758c82f0-8662-11e6-9a5c-4625e5008bf8.png)

I should just give up. It keeps triggering Event One (WHAT THE HECK) and says my other Event does not exist (W H A T  T H E  H E C K). I added a fasting Namespace and everything. **_WHAT AM I DOING WRONG?!_**

**_-AND HOW DID I BREAK THE DECISIONS MODIFIER? I WASN'T EVEN EDITING DECISIONS!!_**

loup99 commented:
First of all, you can't have an even that both is is_triggered_only and has a trigger.

NicolasGrosjean commented:
I have already seen the decisions broken without your changes.
Yesterday, I have fixed several issues on some decisions, maybe it is fixed.

TurtleShroom commented:
@loup99 

Okay. What else is wrong with it?


--------------------------


@NicolasGrosjean 

Good to know. I am glad it wasn't my fault.

TurtleShroom commented:
@loup99  /  @EmperorEnlil 

I can't even order the Event to fire. When I type _"event feelsgoodman.3"_ it says that "event 3 does not exist", and when I type _"event feelsgoodman.1"_ it pulls up Event One (_"event 1"_) and not my Event.

I already put the files in a separate file and assigned _"namespace = feelsgoodman"_ to it. Why won't it recognize it? All of the things I searched said that the game should recognize it, but it WON'T.

loup99 commented:
Your description is the name of the second event.

NicolasGrosjean commented:
A MTTH is missing in your first event.
Like Loup has written in the README.md, I suggest you to use the Validator: it helps a lot to find errors.

TurtleShroom commented:
The Validator doesn't work for me.

loup99 commented:
You are most likely using the wrong settings, wrong path and probably an outdated version.

loup99 commented:
Your frog references are in no way related to the hate symbol called "Pepe", right?

TurtleShroom commented:
Pepe is also used by the mainstream right, Katy Perry, political cartoonists, and the leftists in Tumblr. Anyone who legitimately sees him as a hate symbol is ignorant or has an agenda.

Let me phrase it this way. Hitler loved dogs. Are dogs inherently evil because Hitler used them? No!

The same goes with Pepe. Pepe is a cartoon frog whose use is only limited by one's imagination. The fact that neo-Nazis make use of a cartoon frog does NOT make the frog an irredeemable symbol.

The neo-Nazis did not invent Pepe. Pepe was used for years and only took off because the common man discovered him.

-----------------

Let me make this as clear as possible.

Should Pepe be referenced, he will not, under any circumstances, breach the no-Nazi rules of Paradox Forums, nor will he portray any form of racism or antisemitism. He will be used as he was created to be used, and not for hate.

TurtleShroom commented:
I find the notion that Pepe is a hate symbol to be offensive and demeaning. Pepe is about as much of a hate symbol because neo-Nazis tainted him as the computer is a terroristic device because ISIL uses the Internet to spread their religion of peace. Guilt by association is fallacious.

loup99 commented:
As long as there is no breach of the PI forums rules, there is no issue with making references to frogs. 

Then whether or not "Pepe" is a hate symbol or not is a matter of debate, but according to experts in the domain and his own creator (who criticised the use of the symbol) it is best avoided. 

But the political debate is irrelevant as long as the matter remains in respect of the rules, in both  explicit and implicit ways.

TurtleShroom commented:
Just so we are clear, I reiterate that the Pepes I will share here, both rare and common, are wholesome material that will not breach Paradox rules against racism, antisemitism, and Nazism.

How can this be against Paradox bylaws?
![byzantine pepe](https://cloud.githubusercontent.com/assets/17663470/19215954/5ebebb5a-8d7a-11e6-9192-413da5379833.png)
![et tu pepe](https://cloud.githubusercontent.com/assets/17663470/19215955/6151b14c-8d7a-11e6-8434-3f88682be187.jpg)

I was thinking about the Byzantine Pepe and thought that the exisiting chain where you can subsidize the creation of a holy Icon could have a choice to make one on the cheap by hiring an eccectric man. As a result, he has a chance of creating said Byzantine Pepe as an Icon. You can either tear it down for Piety and for approval of the church, or get a small boost in Culture tech at the cost of angering the Church (identical to the "witch is caught" scenario).

loup99 commented:
Yes, there is no issue with that idea, but do try to use historical pictures for the events.

TurtleShroom commented:
@loup99 

I am still struggling with the Kek Events I added in. I've run out of ideas as to what I can do. Is there anything I am missing?

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -419,7 +419,7 @@ christian = {
 			same_religion_opinion = 25
 			fertility = 0.3
-			add_piety_midifier = -0.25
+			add_piety_midifier = -1
```
Misspelled and too significant.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4855,6 +4867,19 @@ bosporan_group = {
+		defensive_attrition = yes
+		independence_war_score_bonus = 50
+		attacking_same_religion_piety_loss = yes
+		has_heir_designation = yes
```
Why?

loup99 commented on WTWSMS/decisions/conversion_decisions.txt:
```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```
Rulers of holy orders shouldn't convert to local cultures, and this is a vanilla file, so I don't see the need for modifying that.

loup99 replied:
Rulers of holy orders shouldn't convert to local cultures, and this is a vanilla file, so I don't see the need for modifying that.

TurtleShroom replied:
If rulers of Holy Orders don't convert to the local culture, you end up with a French Afghanistan or an Assyrian Mongolia, or a Tangut Russia (three examples of Holy Order cultures). This is a severe oversight on Paradox's part that must be rectified.

Unless you have an alternative means to address this, like making it impossible for Holy Order provinces to flip culture, which would be unrealistic.

loup99 replied:
Given the nature of holy orders the characters culture will switch with time. It makes sense to have holy order = no out of historical reasons.

TurtleShroom replied:
It doesn't because converting a province in Mongolia to France is FAR more unrealistic than a Frenchman not becoming Mongolian after spending his entire life in Mongolia. Culture does not effect his religion.

loup99 replied:
Both are unrealistic. But you are not addressing the issue (culture conversion) by removing the existant checks. I prefer actually addressing it by tweaking the conversion rates.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/decisions/employment_decisions.txt:
```diff
@@ -135,7 +144,10 @@ decisions = {
+						}
 						}
 						add_trait = kshatriya
 						character_event = { id = RoI.30121 }
```
The kshatriya, vaishya and brahmin traits have not been adapted to the Heptal religion, and from my understanding they didn't have castes.

loup99 commented:
Before making any comments, have you fixed your Validator installation?

TurtleShroom commented:
I can't. I'm addition to the ban, I can't even log into the Forum due to errors in their part.

If you can direct me to a mirror to get it, I will get the Validator.

-------

In the Vanilla form, the game auto-converts a sitting Grandmaster to a given culture. I once have the Knights Templar control over Afghanistan, and the entire country became French by game's end.

The reason that Holy Order Grandmasters should be able to convert is to stop that from happening. It doesn't make sense that the Knights Templar would not become more culturally Afghan if it rules Afghanistan for centuries.

-----------

Assyrian Pagans- even unlanded ones -ALL  get a permanent minus one Piety seduction unless at war. Antinomians are not pious.

TurtleShroom commented:
I will revert the caste edits ASAP.

loup99 commented:
- If you can't play the game due to technical issues, you shouldn't use the Validator anyway. No mirror can help you with that. 
- That is unrelated (culture conversion mechanics) and not something we intend on touching now. 
- Why are you talking about the Assyrian religion?

TurtleShroom commented:
1. I can play the game. I cannot view the Forums, and thus can't download the Validator.

2. It's a simple fix and it is more historically accurate than turning a place in India to French.

3. ALL Assyrian characters receive negative one Piety unless at war, making Courtiers in perpetual Piety loss. Precedent exists for a constant negative one Piety loss.

loup99 commented:
- But what were you using above then?
- It doesn't fix the issue, and it makes no sense historically
- But what is the justification for it? It remains misspelled.

TurtleShroom commented:
1. I was manually editing the files. The game works and I got a copy of the Validator before I was banned in June.

2.It injects common sense into the issue and greatly reduces the likelihood of Afghanistan going French (etc.). It is not, nor has it ever been, unrealistic for rulers to assimilate to their realm's culture (or mixing into it). As their is no Afghani-French spring pot, a Grandmaster ruling in Afghanistan for centuries May ultimately become Afghan himself. Excluding Holy Orders from this measure makes absolutely zero sense.

3. Antinomians believed in absolute unrestricted sin if they "asked for forgiveness". How is that not impious? I guess changing what Antinomians call Piety could add to this, since Antinomians do not consider piety to be a lack of sin or resistance to sin like we do. In later games, you can acquire ten units of Piety simply by existing because of your vassals. This is what I am addressing.

loup99 commented:
- Then you should use the Validator with the right path, as I said.
- We don't have any French culture, or Afghanistan in the mod. Furthermore, "common sense" shouldn't be prioritised above historical plausibility. The culture conversion system is already too aggressive. Excluding holy orders is logic, given the non-dynastic nature of them. Knights of French culture joined one holy order, and came from ruling families of Europe. While they may interact with others, their culture didn't change due to the nature of the order. 
- That unpious for non-antinomians

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -481,7 +481,7 @@ christian = {
 		expel_modifier = expelled_jewish
 		
-		general_opinion = -10
+		general_opinion = -5
```
Not going to get into the mod.

TurtleShroom replied:
Okay. Why?

loup99 replied:
Look at the previous commits. I have explained why in those.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -1259,7 +1265,14 @@ muslim = {
 		uses_decadence = no
-		uses_jizya_tax = yes
+		uses_jizya_tax = no
+		can_hold_temples = no
```
Why?

loup99 replied:
What do you mean? I hope that you are aware that the wording "kebab" to refer to Turks is considered as racial slur and not allowed on the Pi forums?

TurtleShroom replied:
See below. This is not the Paradox Forums.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -1527,6 +1540,7 @@ muslim = {
 	}
 	druze = {
 		graphical_culture = muslimgfx
+		has_coa_on_barony_only = yes
```
Why?

TurtleShroom replied:
Druze are not mainstream kebabs. They have very unique beliefs and symbols that don't coincide with Islam. Why should they be using Muslim symbols of that isn't really a thing in their faith?

loup99 replied:
What do you mean? I hope that you are aware that the wording "kebab" to refer to Turks is considered as racial slur and not allowed on the Pi forums?

TurtleShroom replied:
I use "kebab" as a term for all Muslims, and this is not the Paradox Forums. Hence, I will not use that term there. I'm banned anyway.

What I mean is that the Druze are not mainstream and would not use mainstream Islamic symbols when they have their own.

TurtleShroom replied:
I removed "kebab" from my pound sign comment in the Religious text, per your request. Is that what you are referring to?

Moopli replied:
Whether or not the pdox forums allow it does not change its status as a slur.

TurtleShroom commented:
In Vanilla, I have seen Catholic INDIANS join the Knights of Calatrava and MONGOLS and LOMBARDS come to RUN the Teutonic Order.

If we're going to be picky, why not forbid some holy orders from converting (such as the Templars) but not others (such as the Valentinians, Melchisidechians, and others where culture means nothing).

I was using the Knights Templar turning Afghanistan into French culture as an example of the problem.

TurtleShroom commented:
Also, I think that Yazidis, which predate Islam, should start in the Mazden group.

According to the primary (if not official) website of the Yazidi religion (http://www.yeziditruth.org), Yazidis date to the Bronze Age and arose around the time the Jews did.

Also, the chief deity is Melek Taus (Peacock Angel), not Shaytan. Shaytan is either another name for Melek Taus or is a lesser force among the seven angels.

I believe that there may also be a Yazidi province in the Kurdish region. The Kurds have existed for a thousand years and existed by 650 AD, but I am not sure if they existed at this time in history.

TurtleShroom commented:
Perhaps an event tied to the Rise of Islam could move the Yazidis into the Islam group as a Sunni heresy if Kurdistan is conquered?

loup99 commented:
- None of the scenarios that you mention fit the mod timeframe or are relevant to what WtWSMS modifies. I think that is symbolic for why the change is unneeded in addition to what I said earlier which still stands.
- "Mazden"? I assume you mean Mazdan?

TurtleShroom commented:
Yes, I meant Mazdan.

loup99 commented:
On your latest commit:

- Everything from the Heretic Pride mod is going out, as we have no permission. I have explained this before.
- On the changes to k_cyprus this is the explanation for my disagreement:
1. kingdom is titular ->  duchy requirement & independence
2. kingdom is specific -> culture requirement
3. Adding two duplicate requirements which restrict the formation is not necessary if you read the above two points.
4. crusader kingdom -> prestige amount
5. religion -> religious minorities or history have nothing to do with who held the kingdom
- On the changes to k_taurica this is the explanation for my disagreement
1. kingdom is relevant to hellenism -> 600
2. kingdom is relevant but not central for hypsistarianism -> 2000 is  way too high
3. Taurica is a specific Christian title -> no Bosporans
3. Dynamic dejure -> no Bosporans
- I don't get the other weight changes, but the same applies

TurtleShroom commented:
1. I didn't actually employ anything in Heretic Pride. I literally copied and pasted the names of the Seven Angels from Heretic Pride. That's not incorporating the Mod.

2. Ah, so THAT is why the Prestige is so high. Okay, I will revert that.

3. How is Tauricia _that_ relevant to Hellenism, especially if you claim it's a Christian kingdom?

4. I thought that Pontus and Tauricia were the core Hypistarian zones because of their huge Hypistarian majorities. Those two should be the number one goal of Hypistarian conquest.

5. The culture requirement makes no sense. The RL Kingdom of Cyprus was French, and obviously, that's not happening here.

6. I added that because a minority of that size should be able to create the Kingdom if they seize it. If they had such a presence there, why would they not be allowed to become the King of Cyprus if they seized power from the majority religion?

7. How is Taurica a Christian state? That doesn't make any sense, given that you also said that Hellenics want it for some reason. Why should Bosporans not be allowed to create or run the kingdom that they dominate in this era?

8. What does Dynamic De Jure mean?

9. I changed the duchy requirement to Completely Controls so that kings could create Cyprus. Your stipulation was "No Emperors", so your code meant that kings should be allowed to create it.

10.

TurtleShroom commented:
Okay, I reverted the Cypriot Prestige, cut out the Assyrians in Cyprus, removed the use of "kebab" in the text for compliance's sake, undid any references to Heretic Pride, and I changed the Ebionite hatred back to its previous setting.

I am leaving the other stuff for now so I can defend its inclusion through my above replies.

loup99 commented:
- All the scenarios you have mentioned as justifications for your change.
- Then for now we will have to wait.
- As long as the sources back it up it is fine.
- The current is fine
- I'm not the one who designed those. But unless there is some reason for changing it status quo prevails (as with anything, which is why I advised you to focus)
- Hypsistarians
- I don't know who did that, so then for now we will have to wait.
- There are other dejure structures
- Explained below
- Look at the issue for it
- Code in question:
```
	primary_title = {
 				title = d_cyprus
 			}
			independent = yes

```
- All of those related to religion, if possible

TurtleShroom commented:
Okay, your bulleted points are somewhat confusing. I replied above; could you reply to my points in numerical order so I can keep track?

-------------

As for the weights, here's my reasoning:
1. TENGRIS FOR PONTUS: What do the Tengris want with Pontus aside from aggressive expansion? A weight that high implies that something dear to the Tengris is there.

2. ZOROASTRIANS FOR LAZICA: at the beginning of the game, Lazica is Zoroastrian. Naturally, the Zoroastrain empire next door is going to consider it rightful Zoroastrian clay from there on out. However, I gave it a low weight because it is a petty priority.

3. HYPISTARIANS FOR LAZICA: It seems like a reasonable stepping stone between Pontus and Taurica. I was trying to simulate a crescent around the eastern Black Sea stretching from Taurica to Pontus.

4. In the Vanilla rendition, Cumania is one of the largest Tengri regions in the game. In this time period, it is even moreso. I also wanted to add weights to Mongolia and the kingdom to the west of it.

loup99 commented:
All points are in numerical order.

OK to the weights, but could I adapt them to fit with the rest?

TurtleShroom commented:
Absolutely. Adapt them as you see fit. I had rhyme to my reason, but if you insist, do as you will.

TurtleShroom commented:
Adding Autocephaly to the Mimamsas cause them to have _ten_ Pentarchies because of the defaults, plus the actual Mimamsa holy sites. We will need to remove the default Pentarchies and add them back to each religion.

For example, Jerusalem would need to become:

- "pentarchy = nicene"
- "pentarchy = orthodox"
- "pentarchy = miaphysite"

-and so on.

loup99 commented:
- I talked about tuning the numbers, not the locations.
- Yes for the patriarchs, but it will have to wait.
- Why did you remove half of the heads?

TurtleShroom commented:
1. Okay

2. Why will we have to wait on someone to reconfigure the patriarchs one by one? I'd do it if I had the means.

3. I didn't, I merged "WTWSMS_RELIGION_HEADS" with "WTWSMS_PAGAN_RELIGION_HEADS". There were multiple duplicates. If I forgot to merge any head back, please let me know what I missed, and I'll fix it right away.

loup99 commented:
- While all your changes are done out of the blue, when multiple persons work on the same thing, you can't just start changing things that others have been working with and are responsible for. You wanted to redo a system that had just been implemented and now do this.
- I don't know what you added or removed, but you were the one who splitted off some in the first place. As I don't know what is missing or not I will just go back to the original version.

TurtleShroom commented:
1. You said no when I proposed a change to a system; I accepted that change. The same will go with this. Now, specifically, _what did I change that you consider to be in violation_? What if the person responsibile for the prior change never comes? Also, as the owner of the Mod, shouldn't you have the authority to accept any changes?

2. I am not unsure because I already reviewed it. I merged the files together because there were several duplicates. (For example, the religious head for Slavic Paganism was in both files, as was the Aztecs, Baltics, and Zunists.)

loup99 commented:
- Nothing is in violation, I just wan't Enlil to be in board, as he is the mod creator and lead.
- The vanilla heads should preferably not be moved around if not changed.

TurtleShroom commented:
Okay.

loup99 commented:
I removed the duplicates on the main branch.

TurtleShroom commented:
@loup99 

You asked me to re-add the second "WTWSMS_RELIGIOUS_HEADS" file back into the Mod. I did as you asked.

loup99 commented:
Then I ask you to remove all of the moves and adopt the same as the main branch.

TurtleShroom commented:
@loup99 

Gladly. Everything has been updated to the latest incarnation.

loup99 commented:
That was not what I meant (because just as the previous merge you haven't merged all changes, some have instead been overwritten) but fine. The only issue with this is that it will be a lot more tedious.

TurtleShroom commented:
Okay. I'll just leave the Religious Heads as they are right now, so that you can implement the rest of the Mod ASAP.

I've done everything I can to my Kek additions, though, and I'm still not able to get them to work completely. Before implementing my entire Branch, could you take a look at the code and see what I missed?

TurtleShroom commented:
I didn't (and still don't) understand what you asked me to do.

loup99 commented:
What I asked you to do, if you want to, was to write down the changes you want to do about religious heads, holy sites and crusade weights, so that I can directly modify them without having to merge the entire files.

TurtleShroom commented:
Okay. I will wait until TSP VI to mess with any of that. I just want to see this one implemented and my Kek Event checked out.

Why is updating the religious weight and head (etc.) files from the Master such a hassle? Did I overwrite some of my own changes?

loup99 commented:
Are you sure? Then I will ignore what you did, except for the event.

TurtleShroom commented:
You misunderstood. You can drop my changes to Crusade weights and religious heads and implement the rest. With the exception of the Kek events, everything else is ready for full implementation.

loup99 commented on ChangeLog.txt:
```diff
@@ -108,6 +112,8 @@ MINOR:
 - Added Glycerius' assasination attempt in the Nepos startdate
 - Moved Odoacer to Mantua and turned the initial war against Romulus Augustus into a rebellion
+- Powerful Melchisidechians can now restore the Bronze Age Kingdom of Salem and crown themselves the spiritual successor to the Priest-King Melchizidek
+- Tweaked gods and religious flavor for Tzakidite Jews due to their Kabbalism
```
We don't add retro-active changes, so this will not be included.

TurtleShroom replied:
What do you mean? Those changes aren't retroactive, you simply forgot to include them in the Change Log!

loup99 replied:
I only include the changes that are mentioned. :wink: And regardless, the purpose of the changelog is for releases.

loup99 commented on ChangeLog.txt:
```diff
@@ -5,12 +5,15 @@ This release is save-game compatible with previous versions
 - Fixed the issue with the Italian Prefecture having 20 000 in dignity rather than 200, which caused it to stay the primary title even though the holders held empires or kingdoms
 - Modified the Sassanid government type to prevent Zoroastrian characters outside the Sassanids from receiving it
 - Solved the issue which caused temple vassals of the Sassanids to have the wrong holding type
+- Certain Gnostic heresies now use a special flavor image for heretic events rising
```
First of all, what do you mean by this? I think the wording could be clarified. Secondly, as there is no Gnostic group, I think specifying what heresies you speak of would be better.

TurtleShroom replied:
"Certain Gnostic heresies" refer to certain Christian heresies that are Gnostic, not heresies of a Gnostic group.

TurtleShroom replied:
"Certain Gnostic heresies" refer to certain Christian heresies that are Gnostic, not heresies of a Gnostic group.

loup99 replied:
Which heresies?

TurtleShroom replied:
Currently, Marcionism, Melchisedechianism, Sethianism, and Valentinianism will experience this Event.

loup99 replied:
Event picture.

TurtleShroom replied:
Yes, it's an Event Picture.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -1228,7 +1228,11 @@ muslim = {
 	}
 	yazidi = {
-		graphical_culture = muslimgfx
+		ai_convert_same_group = 1 #Yazidis do not normally prolestyze unless asked, they tend to keep their faith to themselves, which is why they are nearing extinction in RL
```
Good idea, but the implementation will change given the new religious system. I will put Yazidi in another category.

TurtleShroom replied:
I'm glad you like my entire Yazidi overhaul. Feel free to ignore the will-convert part if you wish.

loup99 replied:
OK.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4748,7 +4775,6 @@ bosporan_group = {
 	
 	color = { 1.0 1.0 0.68 }
 	
-	# Names given only to Jewish characters (base names)
```
Why did you remove this line? It is just an informing #

TurtleShroom replied:
Hypsistarians are not Jews.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4775,6 +4801,19 @@ bosporan_group = {
+		defensive_attrition = yes
+		independence_war_score_bonus = 50
+		attacking_same_religion_piety_loss = yes
+		has_heir_designation = yes
```
Why?

TurtleShroom replied:
Simple buff for Hypsistarians. The defensive attrition is for obvious reasons (they were difficult to drive out of the Byzantine Empire and it gives them a fighting chance). Attacking the same religion subtracts piety because of the small and close-knit nature of the Hypsistarian faithful. Independence war score bonus simply allows them, like pagans, to break away and protect their faith.

TurtleShroom replied:
Simple buff for Hypsistarians. The defensive attrition is for obvious reasons (they were difficult to drive out of the Byzantine Empire and it gives them a fighting chance). Attacking the same religion subtracts piety because of the small and close-knit nature of the Hypsistarian faithful. Independence war score bonus simply allows them, like pagans, to break away and protect their faith.

loup99 replied:
And heir designation?

loup99 replied:
See above.

TurtleShroom replied:
Hypsistarian tribes seemed to be extremely decentralized.

I personally feel that Hypsistarians should be a faith that can be Reformed. Of course. "bosporan_faith" and "bosporan_faith_reformed" would be in the unique Bosporan Group.

The reason I say this is because there is no collusion between Hypsistarian faithful except for the family and the clans. They were a quiet, close-knit faith reknowned for their honor (I showed you the sources) that we have little information on.

A Reformed Hypsistarian faith would have a centralized, powerful religious head and have conjectural decisions that could be based on the Jewish Decisions.

loup99 replied:
That still doesn't explain the heir designation.

Then I will just leave the Hypsistarians as they are and ask for more research to be made. It is better that we try to make things clear and look at what features they could get.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4936,11 +4975,13 @@ indian_group = {
 	}
 	mimamsa = {
+		has_coa_on_barony_only = yes
+
```
Why?

TurtleShroom replied:
Many of the symbols represent Indian gods and beliefs that did not exist among the Mimamsas, who reject most Hindu deities and focus on the person and the clergy. Also, the entire "replace COA" thing always sucked anyway, but I am not in a place to do that.

TurtleShroom replied:
Many of the symbols represent Indian gods and beliefs that did not exist among the Mimamsas, who reject most Hindu deities and focus on the person and the clergy. Also, the entire "replace COA" thing always sucked anyway, but I am not in a place to change that.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4956,15 +4997,16 @@ indian_group = {
 		}
 		
-		autocephalacy = no
+		autocephaly = yes
```
Will be ignored too, as decided.

loup99 replied:
Yes, we agreed on that, but the issue is that all pentarchies have to redone (which we agree on too)  but I want Enlil tro look at it before.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/traits/02_traits.txt:
```diff
@@ -1,4 +1,4 @@
-########################################################################################################
+C:\Program Files (x86)\Steam\steamapps\common\Crusader Kings II\gfx\traits########################################################################################################
 # WARNING: IF YOU DON'T ADD NEW TRAITS LAST (IN THE LAST READ TRAIT FILE), IT WILL BREAK OLD SAVE GAMES!
```
?

TurtleShroom replied:
That was an accident. I will fix that.

loup99 commented on WTWSMS/common/traits/02_traits.txt:
```diff
@@ -543,6 +547,7 @@ bad_priest_norse = {
 			is_theocracy = yes
 			is_ruler = no
 			controls_religion = norse_pagan_reformed
+			controls_relugion = germanic_pagan_reformed
```
Typo

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/traits/VIET_Immersion_Other.txt:
```diff
@@ -45,7 +46,7 @@ griot = { #West African Griot
 	customizer = no
 	
-	cannot_inherit = yes
+	cannot_inherit = no
```
Why did you change it to no?

TurtleShroom replied:
My research shows that Griots can and do inherit.

TurtleShroom replied:
My research indicated that Griots can and do inherit.

loup99 replied:
During the mod's period, or today? Those traditions can evolve, and the author of those hade done extensive research.

TurtleShroom replied:
I'm not sure if that has always been the case, but all modern Griots can and do inherit. I found nothing indicating that they were like monks and were prohibited to inherit. Griots were basically African bards and historians, not religious figures.

loup99 replied:
Then we will have to do research before determining changes.

TurtleShroom replied:
I think my research is good, but if you want more research, I'll take out this Trait change now so we can implement what has been approved of.

loup99 replied:
What was your source, out of curiosity?

loup99 replied:
I asked for this one too.

loup99 commented on WTWSMS/common/traits/WtWSMS_traits.txt:
```diff
@@ -141,11 +141,14 @@ legendary_leader = {
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
```
That is far too significant...

TurtleShroom replied:
He's the Prophet of Islam. Are you seriously telling me that the Prophet of Islam shouldn't have maximum opinion with all Muslimsn while he is alive? Really?

loup99 replied:
Yes, he shouldn't be a god characters among Muslim characters and have ridiculous martial, going by his life.

TurtleShroom replied:
Fine on ridiculous Martial. but any opinion boost under two hundred is unrealistic. This is the founder of Islam, and all Muslims revere him unquestionably.

loup99 replied:
There are different kinds of opinions though. Even though you have your founder as your liege, you should get upset if he tries to touch crown authority or imprison you for no reason. 25 is largely sufficient.

TurtleShroom replied:
Ah, that makes a lot of sense. Okay, I will lower the opinion bonus further.

loup99 commented on WTWSMS/common/traits/WtWSMS_traits.txt:
```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 	ai_ambition = 1000
+	ai_zeal = 1000
+	ai_honor = -500
+	ai_rationality = 1
```
1000 in ambition is a lot, and why did you add a -500?

TurtleShroom replied:
One thousand ambition was already encoded before I came. One thousand zeal represents the zealous spread of Islam. Negative five hundred honor increases likelihood to break truces, dishonor treaties and alliances, and fight dirty through intrigue, all things that Muhammed totally mastered in his lifetime of conquest.

loup99 replied:
We don't want the AI to break truces though, as that will have consequences similar to what happened with the dignity.

TurtleShroom replied:
-but that is _exactly_ what Muhammed did. He never fought honorably because he was convinced that Allah blessed all that he did, and thus the the ends justified the means. It would be unrealistic to portray him as not resorting to heavy intrigue.

TurtleShroom replied:
If there is an alternate way to give him a propensity to Intrigue, I'm all ears. Maybe adding the "Master Intriguer" Trait will get him to use it more?

loup99 commented on WTWSMS/common/traits/WtWSMS_traits.txt:
```diff
@@ -185,8 +188,15 @@ nasrani = {
 
+	tolerates_christian = yes
+	tolerates_indian_group = yes
+
```
This is not a tolerance trait, those are separate.

TurtleShroom replied:
That was intentional. Nasranis live in India and are constantly exposed to Indian cultures, and vice-versa. Indian religions accept them as an unique caste and even consider them prestigious, as the tooltip and descriptions say.

loup99 commented on WTWSMS/decisions/LI_target_decisions.txt:
```diff
@@ -174,11 +174,18 @@ targetted_decisions = {
 				NOT = { religion = ROOT }
 			}
 
+>>>>>>> refs/remotes/loup99/master
```
Bad merge, discard all of your changes.

TurtleShroom replied:
I don't understand what that means, but I will discard this as soon as I can fix my desktop PC.

loup99 commented on WTWSMS/events/WtWSMS_migration_events.txt:
```diff
@@ -236,14 +236,6 @@ character_event = {
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```
The Migration event is probably the most discussed and reviewed feature. This is staying.

TurtleShroom replied:
Okay.

loup99 replied:
Yes, and that is intended in the **AI Migration Event**.

TurtleShroom replied:
Ah, I was afraid it was a restriction on the player.

I still don't see why we should limit the AI in their migration. They chose to head west in RL, but nothing was stopping them from going east or south. Having them migrate unpredictably would be more realistic than saying where they can migrate.

I'd prefer a means in which they are _more likely_ to migrate west, and if they are Lunatics, they should have a more likely chance _not_ to migrate west. THis way, there is always a chance to go to some bizarre place, but they are most likely to follow the historic route.

loup99 replied:
The easy answer would be: because we decided by consensus. 

The longer answer is that it causes AIs that had settled to migrate, adds randomness (no tribe migrated out of the blue), balkanises the map and goes against historical reasons and conditions behind migration. 

I disagree on Lunatics.

TurtleShroom replied:
Okay.

See, THIS is what a Game Rule would be perfect for.

**[MIGRATION RULE]**
- HISTORIC: Migrating tribes head west and roll over Europe.
- DYNAMIC: Migrating tribes can migrate anywhere in range.

That way, players can decide how they want the game to look. I'd code it myself if I could. It's all about choice.

TurtleShroom replied:
@loup99 / @EmperorEnlil 

Okay.

See, THIS is what a Game Rule would be perfect for.

**[MIGRATION RULE]**
- HISTORIC: Migrating tribes head west and roll over Europe.
- DYNAMIC: Migrating tribes can migrate anywhere in range.

That way, players can decide how they want the game to look. I'd code it myself if I could. It's all about choice.

TurtleShroom replied:
Okay. I won't touch the mechanic without explicit permission from here on out.

-but you SERIOUSLY need to implement a Rule to allow for random Migration. Watching Norsemen show up in Turkestan is exciting and dynamic to me. You are right to say it is ahistorical, but you shouldn't shut it down because of it. The Migration mechanic is the biggest feature of this Mod and we should have the choice on how crazy it gets.

loup99 replied:
Customisation is not something I'm opposed to, but it isn't a priority. Do remember that the more options we add, the more weights and setting we have to maintain.

TurtleShroom replied:
@loup99 / @EmperorEnlil 

Actually, I don't think it will!

It may only need lines in one file. Where it shows the AI target, you would put

AND = {
has_game_rule = {
name = MIGRATION
value = HISTORICAL
}
OR = {
(usual targets)
}
}

loup99 replied:
No, that is not what we have planned, so no, it is far more complicated.

loup99 commented on WTWSMS/events/WtWSMS_schisms.txt:
```diff
@@ -271,6 +271,22 @@ character_event = {
+		set_parent_religion = { religion = psilanthropism parent = orthodox }
+		set_parent_religion = { religion = priscillianism parent = zoroastrian } #This is basically Maneachism, and in fact, it should be a heresy of that already
+		
+		
```
Moving around many heresies in an event that will be redone, without factoring in the schism in question, won't work. Furthermore, your motivation being "should already be in this group" does not work

TurtleShroom replied:
I did factor in the schism they became heresies of. With the exception of Priscillianism, I researched each one and determined the denomination most like it.
Several heresies were left behind by mistake. I did something that NEEDED to be done!

However, if it makes you happy, I will remove Priscillianism.

loup99 replied:
I think there was a misunderstanding: the idea with the schism is that heresies can become different depending on the outcome. It may need to be done, _once the event is enabled_ but Priscillianism should definitely not move to the category you assigned it.

loup99 replied:
That is the entire plan since start, yes. It would be helpful if you tried to follow development, or ask.

The current event is disabled anyway.

loup99 replied:
There is no plan to make it into a prototype as far as I know, but yes, your changes make sense to keep (except for the one we discussed).

TurtleShroom replied:
Got it. My initial choice of Prisicllianism was a severe mistake. Priscillians were VERY Christian indeed, and they actually split from the Catholic Church. They will now move into the Catholic group at the Schism.

Moopli commented:
So uh, what't with the kek events, and pepe, etc?

Reminds me of the funny events in VIET which everyone grows tired of almost immediately. What do they actually add to the mod?

TurtleShroom commented:
@Moopli

Are you REALLY going to inject "NO FUN ALLOWED" into this Mod? The Kek Event fired in a very specific setting and on a very specific set of characters, and only one one religion while travelling. It is not spam.

-and might I add, I haven't seen many VIET humor events outside the code, presumably because people like those you described loathe humor and so ruin it for the rest of us.

Regarding the Pepe jokes and frog memes, the only event I have planned is an early childhood event that has a chance to add friendship and upbringing traits.

No event or meme joke is going to be a common, constant Event like the "I found my ring" one.

TurtleShroom commented:
I'm so sorry for the double posts. My computer froze up and I didn't get the message that the comments were posted, so I resubmitted them.

loup99 commented:
Concerning the "fun" events:
- It is not a priority to add such events (the VIET ones should be disabled after some of them having been in the mod)
- If the event is considered relevant and adds to an existing mechanic, my personal opinion is that it isn't an issue. 
- And no, this isn't about loathing fun or not, it is to find a fair middle-ground in which relevant events are kept and just "fun" removed.

TurtleShroom commented:
An Event is an Event is an Event is an Event. Assuming the Event isn't a constant pop-up spam and only fires in select circumstances, why shouldn't it be in the game? The same goes with all the VIET humor events.
The Kek Event is a prime example: it fires only for Egyptian Pagan rulers, and only while Travelling, which means players will not encounter it unless they meet those strict requirements, and even then, only if they choose to.

Moopli commented:
> "NO FUN ALLOWED"

Chill. What I want to see is that events which try to be funny actually succeed at it. Maybe it's a question of the 'authorial voice' not working well, maybe it's a question about timing, but it's easy for jokes to fall flat.

For example, consider an event where you discover an old church containing a mosaic that depicts pepe bestowing his blessing upon a true believer. It could just randomly pop up, probably with restrictions on who can get it; or it could be a part of an event chain, where the events along the chain manager to build up some jokes, and the pepe mosaic is more of a punchline than just a random "lol look it's pepe".

That can help you handle timing, building up to a joke. As for 'authorial voice', some jokes work better in certain formats than others; and reading your joke events I just felt a resounding meh. I do like absurd humour, but it's hard -- you have to create a scenario that juxtaposes the ridiculous with the mundane, and that usually requires much more persistence than a simple two-piece joke. Something much easier to pull off in CK2's confines is dry wit or sardonic humour -- for example, a two-event chain where your maidservant is mopping the floor, you send her off to tend to your child before she has a chance to dry the floor. Next event, your hated spouse runs by as you work, slips, and falls, and the only button you can click says "Oops. [5% spouse dies, 5% spouse incapable, 10%Spouse Wounded, Spouse temporary -20 opinion of you]".

Then again, you seem to think your joke events are funny, so maybe there are other devs who agree.

TurtleShroom commented:
The Kek joke was an attempt at subtlety. I did not overtly refer to the "PRAISE KEK" meme, but for those who know of it, they can see it. For everyone else, it's just "someone did their research". I made the effort to research what Kek actually stood for in RL and tried to make it so that people who did not recognize the "meme magic" would nonetheless enjoy a flavorful rendition of worshipping an actual god the Egyptians worshipped.

As for the Pepe Icon joke, as I noted, you had the option to pay for a real Icon at full price, or pay an eccentric lunatic to draw one up on the cheap. Pepe has a random chance of showing up under that choice, but it's not a guarentee, which limits the times you'll see him and prevents spam.

Moopli commented:
Alright, well, I'm not gonna fight it, but I still think the jokes are just 'eh', so if you're going to pull the self-righteous "NO FUN ALLOWED?" card then you could try to make funnier jokes.

TurtleShroom commented:
I didn't say "NO FUN ALLOWED" because you didn't think my jokes were funny. I said it because I thought you were pushing to prohibit the inclusion of memes, jokes, and meme jokes, even if I made the effort to adjust them to fit this time period.

Moopli commented:
Alright, sorry, a dickish thing for me to say. Luckily I think we clarified my stance.

My word isn't law though anyway so I guess it's up to loup :p

TurtleShroom commented:
Heh, I accidently caused the Great Schism to fire at the start of the game. Whoops; I'm fixing that now.

loup99 commented:
As I don't get the joke I'm not the right person to discuss it being included or not.

loup99 commented:
Concerning the events, given that Erik is against their addition, and Enlil hasn't given his opinion, I think the integration of them will have to wait upon that.

TurtleShroom commented:
Moopli seems okay with them. I don't know why you wouldn't approve of them. If you don't get the joke, it's just a clever flavor addition for an actual Egyptian god.

loup99 commented:
If I don't know, Moopli said his opinion above, Erik is against, Enlil hasn't made a statement and you are in favour, it will have to wait as there is no clear majority or common stance.

TurtleShroom commented:
@loup99 

Okay. I'll just put them off, then. Are you ready to implement all my other changes? We've already cleared them all, remember, except for the Hypsistarians' heir designation, which I have disabled pending extensive further research.

loup99 commented:
I haven't reviewed everything, and this time I will have to do extensive validation and testing of everything.

TurtleShroom commented:
@loup99

Ain't nobody got time for that.

I will revert the Muhammad content so that we can FINALLY implement this Branch. I have waited long enough.

loup99 commented:
No, I don't really have it, so it will take a while, if that was what you referred to.

TurtleShroom commented:
@loup99 

Do you mean there is something preventing the edit from being Committed?

If it's blocking implementation, I'll disable the frog events for now.

TurtleShroom commented:
I have personally tested the Decisions that restore Salem for Melchisedechians and that elevate the Bishop of Avila for Priscillians.

loup99 commented:
No, nothing specific, just the standard testing process.

TurtleShroom commented:
Okay. I won't be making any major changes from here until implementation.

loup99 commented:
> 2. As an African sect, Donatists now use African GFX
Isn't this going to make them have relatively weird GFX? They are still Christian.

TurtleShroom commented:
No, you are confusing the religious GFX with the GFX shields on the map. Priscillianism uses Muslim GFX, which means rhombuses for realm shields. It won't change the main graphics of the faith or it's dynastic COAs.

loup99 commented:
Why would they use Muslim GFX?

TurtleShroom commented:
-because "Muslim" GFX is better described as "Middle Eastern" GFX. As I wrote in the comments, Marcus (the doctrinal father of Priscillianism) was from Egypt. It also makes it look unique.

loup99 commented:
The movement itself was from Iberia, so it doesn't really fit.

TurtleShroom commented:
Hello. I am writing today to ask how the implementation of my Branch is going along. It's been a while, and I have ceased updating it pending implementation. Is there any news on this front?

loup99 commented:
No. Do you have anything to answer to the previous comment?

TurtleShroom commented:
I gave my reason and I think it looks nice. I'm fine with reverting it if you want.

loup99 commented:
If that is the only motivation I disagree. But I don't know what the others think.

TurtleShroom commented:
Okay. I will revert it now.

Is there any word on implementation?

loup99 commented:
Nothing more than I said previously.

TurtleShroom commented:
Okay, how about now? Is there any word on implementation?

loup99 commented:
Don't worry, I will tell you when there is.

loup99 commented:
We don't use the vanilla localisation files. All lines that we replace are in 00_WtWSMS_override.csv. This is a question of compatibility and maintainability. 

As for the investiture, I can't see what you changed, but the same applies. The vanilla system is sufficient in that domain, and as far as I know no non-vanilla religion in the mod has investiture.

Concerning the modifier, you just disabled the modifiers, that isn't balanced and the whole chain shouldn't have duplicate religion requirements.

With your additions, there are more than five priscillian holy sites. Furthermore, you said they were "nonsensical", but didn't explain why?

TurtleShroom commented:
I made further changes after that. I just haven't Committed them yet.

Aquelians, Donatists, and Priscillians all use Investiture (and the former two have had it before I came on board).

The "decadent Christian" Scripted Trigger Modifier was broken. The pound signs I added were SOLELY for that Modifier, which did not break the chain.

Priscilliams revering Antioch and Constsntinople is nonsensical because of its irrelevancy to that faith. Priscillian was in Iberia and Occitania and tried in Rome. Marcus' approximate birthplace rounds out the list.As I mentioned earlier, Marcus was  born in Egypt.

TurtleShroom commented:
I will fix the localization immediately. I did not know that, so I will change it.

loup99 commented:
For performance, I removed the modifiers.
